src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -1,9 +1,42 @@
-<Project Sdk="Microsoft.Build.NoTargets">
+<Project Sdk="Microsoft.NET.SDK">
   <!-- These are wrapper project files for packaging.-->
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0</TargetFrameworks>
-    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
-    <!-- Avoid trying to binplace non existent documentation file.-->
-    <EnableBinPlacing>false</EnableBinPlacing>
+    <TargetFrameworks>$(NetCoreAppToolCurrent);net472</TargetFrameworks>
+    <PackageId>$(MSBuildProjectName)</PackageId>
+    <AvoidRestoreCycleOnSelfReference>true</AvoidRestoreCycleOnSelfReference>
+    <AssemblyName>Microsoft.NETCore.Platforms.BuildTasks</AssemblyName>
+
+    <IsSourceProject>false</IsSourceProject>
+    <IsPackable>true</IsPackable>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
```

Comment:
I'm not sure I follow what is the purpose of bringing the GenerateRuntimeGraph task over if it is not being packaged or used here. Is the plan in the future to remove the checked in runtime graph and instead generate it as part of the build using the task? I see that perhaps the plan is to keep the task that generates the files right next to the package that ships them, but if this doesn't change often, I wonder why not to keep the task in Arcade. BTW I'm not opposing to the change, I just want to understand better the motive.

Reply:
Nvm, I see now how we use this now in runtimeGroups.props to generate the final runtimejson.

Reply:
> if it is not being packaged or used here

It is used here.  https://github.com/dotnet/runtime/blob/7b0075d19ecc1ce6c1bcf8a8171601f18b2e1117/src/libraries/Microsoft.NETCore.Platforms/pkg/runtimeGroups.props#L265-L275

> perhaps the plan is to keep the task that generates the files right next to the package that ships them, but if this doesn't change often, I wonder why not to keep the task in Arcade

See https://github.com/dotnet/runtime/issues/48507
We're planning to run the task during source build to automatically add more RIDs to the graph.
Also see https://github.com/dotnet/arcade/pull/7141 and https://github.com/dotnet/runtime/pull/50157